ctru-rs/src/mii.rs

```diff
@@ -0,0 +1,435 @@
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum RegionLock {
+    NoLock,
+    Japan,
+    USA,
+    Europe,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum Charset {
+    JapanUSAEurope,
+    China,
+    Korea,
+    Taiwan,
+}
+
+#[derive(Clone, Debug)]
+pub struct MiiDataOptions {
+    pub is_copying_allowed: bool,
+    pub is_profanity_flag_enabled: bool,
+    pub region_lock: RegionLock,
+    pub charset: Charset,
+}
+
+#[derive(Clone, Debug)]
+pub struct SelectorPosition {
+    pub page_index: u8,
+    pub slot_index: u8,
+}
+
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub enum OriginConsole {
+    ConsoleWii,
```

Comment:
Remove the `Console` prefix from all variants.

Reply:
But Rust doesn't allow numerals at the start of a member (3DS)

Reply:
True, hadn't noticed... Still, does "WiiUSwitch" actually exist as an option? I don't see it mentioned in `libctru`'s docs.

Reply:
The 3dbrew wiki says so

Reply:
You can use `N3DS` instead

Reply:
> You can use N3DS instead

Just to clarify (_sorry it is late here_), ``N3DS`` instead of ``WiiUSwitch`` right?

Reply:
No, write `N3DS` instead of `Console3DS`. At that point you can remove all the `Console` prefixes.

Reply:
Alright, that makes sense, idk why I thought that it was meant to replace ``WiiUSwitch``.
About replacing ``Console3DS`` with ``N3DS``, wouldn't people confuse ``N3DS`` with New 3DS?

Reply:
We can use `New3DS` or `NewN3DS` for that. `N3DS` could mean "Nintendo 3DS"

Reply:
Or use `O3DS` for "Original 3DS", but I don't think that's used often.

I named one of my repos [`n3ds-controller`](https://github.com/AzureMarker/n3ds-controller/) because of this issue.

Reply:
I think I'll use N3DS and clarify this in the documentation, seems like the best option to me